appshell/appshell_extensions_platform.h

```diff
@@ -97,6 +98,21 @@ int32 ShowSaveDialog(ExtensionString title,
                        ExtensionString initialDirectory,
                        ExtensionString proposedNewFilename,
                        ExtensionString& newFilePath);
+#else
+void ShowOpenDialog(bool allowMulitpleSelection,
```

Comment:
I'm of the opinion that we should try using unified signatures as much as possible.